src/responder/common/negcache_files.c

```diff
@@ -19,94 +19,86 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <sys/types.h>
-#include <pwd.h>
-#include <grp.h>
 #include "util/util.h"
+#include "util/nss_dl_load.h"
 #include "responder/common/negcache_files.h"
 
 #define BUFFER_SIZE 16384
 
-bool is_user_local_by_name(const char *name)
+bool is_user_local_by_name(const struct sss_nss_ops *ops, const char *name)
 {
     struct passwd pwd = { 0 };
-    struct passwd *pwd_result;
+    int errnop;
     char buffer[BUFFER_SIZE];
-    bool is_local = false;
-    int ret;
+    enum nss_status ret;
     char *shortname = NULL;
 
-    ret = sss_parse_internal_fqname(NULL, name, &shortname, NULL);
-    if (ret != EOK) {
+    if (sss_parse_internal_fqname(NULL, name, &shortname, NULL) != EOK) {
```

Comment:
Please keep the original coding style:
```
ret = foo()
if (ret != EOK)
```
This allows to examine the return code more easily in debugger.

Reply:
Fixed.